### PR TITLE
Web content

### PR DESCRIPTION
**Pull request by ommarz**
Added files.

### CODE REVIEW

File: README.md
Comment (ommarz) [inline README.md:1]: You are doing a great job so far!  Please kindly review the following comments to make it even better:

Please remember that a README.md file is an important file to document and let know the community what is it about and how it works. Please use the [Microverse template](https://github.com/microverseinc/readme-template) and customize it according to your project.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+}
+form{
+  width: 640px;
+  background-color: rgb(39, 38, 69);
+  color:white;
+  margin: 0 auto;
+  padding: 30px;
+}
+#name, #email, #number, select{
```
Comment (ommarz) [inline css/style.css:18]: Nice job with your styling! Please remember that if more than 1 ids need the same styling, you can always create a class for all of the elements to only create one styling rule. Please make the changes accordingly.

**Review by ommarz — COMMENTED** [review]
Excellent job on your project so far! Please take a look at the comments below to make it even more professional.

Remember that when we commit to a project, the title and message of the commit are instructions that will have an effect during the commit, so please write them in infinitives and the most clear and descriptive as possible.  Take a look at the ['Document your repo in a professional way' article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md) if you need more clarification on this.

**Review by AbelG101 — APPROVED** [review]
# Hi @ommarz 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Proper use of the "Review changes" feature from GitHub. ✔
- Each comment has a proper explanation ✔
- Proper use of GitHub inline comments feature ✔
- Each comment is phrased politely ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**